src/irmin-pack/s.ml

```diff
@@ -21,3 +21,188 @@ module type ATOMIC_WRITE_STORE = sig
 
   val flush : t -> unit
 end
+
+module type LAYERED_CONTENT_ADDRESSABLE_STORE = sig
+  include Pack.S
+
+  module U : Pack.S
+
+  module L : Pack.S
+
+  val v :
+    [ `Read ] U.t ->
+    [ `Read ] U.t ->
+    [ `Read ] L.t option ->
```

Comment:
It's because I'm using the function `val v : Conf.t -> [ `Read ] t Lwt.t` (https://github.com/mirage/irmin/blob/master/src/irmin/s.ml) to build the stores from which I then build a layered store.

Reply:
Hmm, that type looks broken to me too, but it's not related to this PR.